docs/guide/dev/integration/angular/README.md

```diff
@@ -197,6 +197,30 @@ This property takes precedence over the {@linkapi CKEDITOR.config#readOnly `conf
 
 The following `@Output` properties are supported by the CKEditor 4 Angular component:
 
+### `namespaceLoaded`
+
+Fired when the {@linkapi CKEDITOR `CKEDITOR` namespace} is loaded. It only triggers once, no matter how many CKEditor 4 components are initialised. Can be used for convenient changes in the namespace, e.g. for adding external plugins:
```

Comment:
```suggestion
Fires when the {@linkapi CKEDITOR `CKEDITOR` namespace} is loaded. It only triggers once, no matter how many CKEditor 4 components are initialised. Can be used for convenient changes in the namespace, e.g. for adding external plugins:
```

:point\_up: Just keeping it consistent with the rest of the event descriptions which use present tense.